pkg/schedulingtypes/typeregistry.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
How about move this to `schedulingtypes/utils`?

Reply:
This is not a utility file, but inherent component of the main functionality.